Update laser examples (#1378)
chaos-polymtl/lethe: doc/source/examples/multiphysics/static-irradiation/static-irradiation.rst

| :math:`H`                 | :math:`170\;\mu\text{m}`  | :math:`\vec{u}_{\text{in}}`| :math:`0.1\;\text{m s}^{-1}`|
+---------------------------+---------------------------+----------------------------+-----------------------------+
| :math:`\Delta h`          | :math:`20\;\mu\text{m}`   | :math:`T_{\text{in}}`      | :math:`298\;\text{K}`       |
+---------------------------+---------------------------+----------------------------+-----------------------------+
| :math:`L`                 | :math:`600\;\mu\text{m}`  | :math:`T_{\text{0}}`       | :math:`298\;\text{K}`       |
+---------------------------+---------------------------+----------------------------+-----------------------------+

There are three phases involved in this simulation: solid and liquid Ti6Al4V, and Argon. The metal-gas interface is handled by the VOF solver, while the solid-liquid interface is obtained from the temperature field of the HT solver. Hence, this example models a two-fluid problem: ``fluid 0`` corresponds to the Argon phase and ``fluid 1`` is the metal (solid and liquid), for which the solid part corresponds to a infinitely viscous fluid. 

.. note::

>   To improve the performances of the solvers, all dimensional quantites in this example are based on the SI system except for the reference length, which is taken as :math:`1\;\text{mm}`.

Reviewer: I would give a quick explanation as to why this improves performance
```suggestion
  To improve the performance of the solvers, all dimensional quantities in this example are based on the SI system except for the reference length, which is taken as :math:`1\;\text{mm}`.
```

    
--------------
Parameter File
--------------

Simulation Control
~~~~~~~~~~~~~~~~~~

> The time integration is handled by a 2nd-order backward differentiation scheme (``bdf1``) with a maximum time-step of :math:`\Delta t = 1.9 \times 10^{-8} \; \text{s} < \Delta t_\sigma` which corresponds to the capillary time-step constraint (see :doc:`capillary wave example <../capillary-wave/capillary-wave>`). We use adaptive time stepping with a maximum CFL of :math:`0.06` to prevent instability resulting from the explicit coupling between the NS and HT solvers through the recoil pressure and evaporative cooling. 

Reviewer: ```suggestion
The time integration is handled by a first order backward differentiation scheme (``bdf1``) with a maximum time-step of :math:`\Delta t = 1.9 \times 10^{-8} \; \text{s} < \Delta t_\sigma` which corresponds to the capillary time-step constraint (see :doc:`capillary wave example <../capillary-wave/capillary-wave>`). We use adaptive time stepping with a maximum CFL of :math:`0.06` to prevent instability resulting from the explicit coupling between the NS and HT solvers through the recoil pressure and evaporative cooling. 
```

We also set in this subsection the reference surface tension coefficient of the metal-gas interface and its temperature derivative to simulate the Maragoni effect. Here, we consider a linear evolution of the surface tension coefficient with the temperature at the liquid-gas interface, and we neglect its effect at the solid-gas interface to avoid numerical instabilities. This is done by setting ``surface tension model = phase change``. We refer to the parameter guide :doc:`../../../../parameters/cfd/physical_properties` for more details on this model.

.. note::

>   To improve the performances of the solvers, all dimensional quantites below are based on the SI system except for the reference length, which is taken as :math:`1\;\text{mm}`.

Reviewer: Do you need to repeat this?

.. math::

    q_\text{evap} = \phi_\text{evap} L_\text{vap} p_\text{sat}\sqrt{\frac{M}{2\pi R}}

.. math::

    p_\text{rec} = \psi_\text{evap} p_\text{sat}

where :math:`\phi_\text{evap}=0.82` and :math:`\psi_\text{evap}=0.56` are the ``evaporation coefficient`` and ``recoil pressure coefficient``, respectively, :math:`L_\text{vap}=8.9\times 10^{6}\;\text{Jkg}^{-1}` is the ``evaporation latent heat``, :math:`M=4.58\times 10^{-2}` is the ``molar mass`` of the metal, :math:`R=8.314\;\text{Jmol K}^{-1}` is the ``universal gas constant`` and :math:`p_\text{sat}` is the saturation pressure. The latter is computed according to:

.. math::

    p_\text{sat} = p_\text{atm}\exp{\left[\frac{L_\text{vap}M}{RT_\text{boil}}\left(1-\frac{T_\text{boil}}{T}\right)\right]}

where :math:`p_\text{atm}=101.325\;\text{kPa}` is the ``ambient pressure``, and :math:`T_\text{boil}=3550\;\text{K}` is the ``boiling temperature``.

Both terms are then applied at the liquid-gas interface using the Continuous Surface Force (CSF) model, as described for the surface tension in :doc:`../../../theory/multiphase/cfd/vof` theory guide.

    
Non-Linear Solver
~~~~~~~~~~~~~~~~~

> The parameters for the non-linear system resolution of the three physiscs are set in the ``non linear solver`` subsection.

Reviewer: ```suggestion
The parameters for the non-linear system resolution of the three physiscs are set in the ``non-linear solver`` subsection.
```

.. code-block:: text

    --------------
    Heat Transfer
    --------------
    Newton iteration: 0  - Residual:  1.985e+07
      -Tolerance of iterative solver is : 1.985e+05
      -Iterative solver took : 2 steps to reach a residual norm of 3.944e+04
    	alpha =      1 res = 1.583e+05	||dT||_L2 =  71.89	||dT||_Linfty = 15.46
    Newton iteration: 1  - Residual:  1.583e+05
      -Tolerance of iterative solver is : 1583
      -Iterative solver took : 2 steps to reach a residual norm of 409.8
    	alpha =      1 res =   1074	||dT||_L2 = 0.3355	||dT||_Linfty = 0.103
    Newton iteration: 2  - Residual:  1074
      -Tolerance of iterative solver is : 10.74
      -Iterative solver took : 2 steps to reach a residual norm of 5.567
    	alpha =      1 res =   5.47	||dT||_L2 = 0.0111	||dT||_Linfty = 0.001807

The linear solver tolerances are set accordingly.

****

-----------------------
Running the Simulation
-----------------------

> We call ``lethe-fluid`` to launch the simulation by invoking the following command:

Reviewer 2: ```suggestion
We call ``lethe-fluid`` to launch the simulation by invoking the following command from the ``2D`` subdirectory:
```
